src/components/AeList.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <ul class="aeList">
+    <slot/>
+  </ul>
+</template>
+
+<script>
+export default {
+  name: 'AeList',
+  props: {items: {type: Array, default: () => []}}
+}
+</script>
+
+<style scoped>
+  ul {
```

Comment:
For performance reasons, all styles of a component should be scoped with the class of the root node, in this case:
```
.ae-list {
  list-style: none;
  padding-left: 0;
}
```
https://vue-loader-v14.vuejs.org/en/features/scoped-css.html#also-keep-in-mind